contracts/Staking.sol

```diff
@@ -0,0 +1,205 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "hardhat/console.sol";
+
+contract Staking is ERC721URIStorage, Ownable {
```

Comment:
maybe we won't need this ownership based stuff if we're not planning to change owner

Reply:
@therealdannzor -> I've removed the library now, can you have a look through and check the new modifier is acceptable.